manager-bundle/src/Command/InstallBinDirCommand.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of Contao.
+ *
+ * (c) Leo Feyer
+ *
+ * @license LGPL-3.0-or-later
+ */
+
+namespace Contao\ManagerBundle\Command;
+
+use Symfony\Component\Console\Attribute\AsCommand;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Style\SymfonyStyle;
+use Symfony\Component\Filesystem\Filesystem;
+use Symfony\Component\Filesystem\Path;
+
+#[AsCommand(
+    name: 'contao:install-bin-dir',
+    description: 'Installs the bin/console file.'
+)]
+class InstallBinDirCommand extends Command
+{
+    public function __construct(private string $projectDir)
+    {
+        parent::__construct();
+    }
+
+    protected function execute(InputInterface $input, OutputInterface $output): int
+    {
+        $fs = new Filesystem();
+        $io = new SymfonyStyle($input, $output);
+
+        $sourcePath = __DIR__.'/../../skeleton/bin/console';
```

Comment:
```suggestion
        $sourcePath = Path::canonicalize(__DIR__.'/../../skeleton/bin/console');
```

Reply:
I have copied this from the `InstallWebDirCommand` where it seems to work without `Path::canonicalize()`. So either we change it there as well or we leave it as it is.

Reply:
I would vote for using it in both places, but feel free to resolve this discussion anyways. :upside_down_face:

Reply:
Changed in b5eb0f0c988bdc69e7120e50861cb2bd98dcba8c.